manage.py

```diff
@@ -29,11 +29,11 @@ def list_routes():
 
         methods = ','.join(rule.methods)
         url = url_for(rule.endpoint, **options)
-        line = urllib.unquote("{:50s} {:20s} {}".format(rule.endpoint, methods, url))
+        line = urllib.parse.unquote("{:50s} {:20s} {}".format(rule.endpoint, methods, url))
         output.append(line)
 
     for line in sorted(output):
-        print line
+        print(line)
```

Comment:
Do we really need this for loop?

Reply:
Probably not, but the purpose of this commit is to upgrade to Python 3.  We can remove in a separate commit if need be.